app/views/tweets/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h4>Spread the word!</h4>
+<small>Powered by Twitter</small>
+<hr>
+<% if !current_user.uid.nil? %>
+<p>
+  <%= form_for :tweet, url: tweets_path, method: :post do |f| %>
+  <%= f.text_field :message %>
```

Comment:
Is this presentation clear visually, or should there be some kind of text label?